airbyte-cdk/bulk/core/load/build.gradle

```diff
@@ -10,3 +23,18 @@ dependencies {
     testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.8.1")
     implementation "org.jetbrains.kotlin:kotlin-reflect:2.0.20"
 }
+
+task integrationTest(type: Test) {
+    description = 'Runs the integration tests.'
+    group = 'verification'
+    testClassesDirs = sourceSets.integrationTest.output.classesDirs
+    classpath = sourceSets.integrationTest.runtimeClasspath
+    useJUnitPlatform()
+    mustRunAfter tasks.check
```

Comment:
what's this for? why not have this run as part of `check`? is this because of the dependency on `assemble` for the docker image? if it's the latter, it's better to declare that dependency explicitly with `dependsOn assemble`. These `mustRunAfter` constraints are typically not what you really want for these kinds of tasks.

Reply:
this really tripped me up; did you in fact mean to have `check` depend on `integrationTest`? if so please let me know @edgao

Reply:
iirc I copypasted this from stackoverflow without reading it :P I did in fact want to have check depend on integrationTest

(which I think is what I did later on, with the `rootProject.check.dependsOn` thing? This mustRunAfter thing probably isn't really needed - probably the assumption on SO was that integrationTest is slow, and therefore only worth running if `check` succeeded. Which isn't true here.)

Reply:
Thanks!

FYI chatgpt is great at generating gradle scripts